### ISSUE TITLE
[A11y][SF][MessageView]: [Bug Description]

### ISSUE THREAD

**Issue by tandan0813** [SAP/ui5-webcomponents-react#7748]
### Describe the bug

It contains Lists in `MessageView` component.

However, there is a violation that in lists screen reader users are not informed about pressing f2.This was introduced with https://jira.tools.sap/browse/SAPDESIGNSYSTEM-91 .
Compare with https://snippix.only.sap/snippets/394357

<img width="1761" height="496" alt="Image" src="https://github.com/user-attachments/assets/331c02ea-d229-42a4-90f8-b404fc385c0f" />

Please remain the same announcement.

### Reproduction steps

1.
2.
3.
...

### UI5 Web Components for React Version

2.13.2

### UI5 Web Components Version

2.13.2

### Browser

Chrome

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @tandan0813 

Please follow the guidelines for opening issues. "[MessageView]: [Bug Description]" doesn't give information of what the issue is about.

The `MessageView` implements the `ui5-list` (`List`) web component. So, once the announcement is available in the `List` it's also available in the `MessageView`. As there is no action required on our end, I'm going to close this issue.
